pkg/api/types.go

```diff
@@ -153,6 +153,15 @@ const (
 	TerminationMessagePathDefault string = "/dev/termination-log"
 )
 
+// There is no namespace required for these kinds specified here.
+// If a kind is not enumerated here, it is assumed to have a namespace scope.
+var NoneNamespacedKinds = []string{
```

Comment:
OK, I'll look into the instance @smarterclayton told, then I'll try to use the similar one for this PR.